rts/System/Platform/Threading.cpp

```diff
@@ -31,6 +33,42 @@
 #include "System/Misc/TracyDefs.h"
 
 namespace Threading {
+
+enum ConfigPinPolicy {
+	None,
+	SystemDefault,
+	ExclusivePerformanceCore,
+	SharedPerformanceCores,
+
+	MinimumValue = None,
+	MaximumValue = SharedPerformanceCores,
+};
+
+CONFIG(int, ThreadPinPolicy)
+	.defaultValue(ConfigPinPolicy::SystemDefault)
+	.safemodeValue(ConfigPinPolicy::None)
+	.minimumValue(ConfigPinPolicy::MinimumValue)
+	.maximumValue(ConfigPinPolicy::MaximumValue)
+	.description("Thread to CPU Pinning Policy (0) = Off; (1) = System Default; (2) = Exclusive Performance Core; (3) = Share Performance Cores");
+
+
+	cpu_topology::ThreadPinPolicy GetChosenThreadPinPolicy() {
+		int configPinPolicy = configHandler->GetInt("ThreadPinPolicy");
+		switch (configPinPolicy) {
+			case ConfigPinPolicy::None:
+				return cpu_topology::THREAD_PIN_POLICY_NONE;
+			case ConfigPinPolicy::ExclusivePerformanceCore:
+				return cpu_topology::THREAD_PIN_POLICY_PER_PERF_CORE;
+			case ConfigPinPolicy::SharedPerformanceCores:
+				return cpu_topology::THREAD_PIN_POLICY_ANY_PERF_CORE;
+			
+			// ConfigPinPolicy::SystemDefault jumps here as well.
+			default:
```

Comment:
Consider making it explicit, basically same readability either way:
```suggestion
			case ConfigPinPolicy::SystemDefault:
			default:
```

Reply:
Done.